deepmerge/strategy/core.py

```diff
@@ -1,20 +1,33 @@
+from __future__ import annotations
+
+import sys
+from typing import Callable, Any
+
+if sys.version_info >= (3, 10):
+    from typing import TypeAlias
+else:
+    from typing_extentions import TypeAlias
```

Comment:
```suggestion
    from typing_extensions import TypeAlias
```
I think this is what's causing 3.8/3.9 to fail.